package/cfg/k3s-cis-1.8-hardened/config.yaml

```
@@ -0,0 +1,53 @@
+---
+## Version-specific settings that override the values in cfg/config.yaml
+
+master:
+  components:
+    - apiserver
+    - kubelet
+    - scheduler
+    - controllermanager
+    - etcd
+    - policies
+
+  apiserver:
+    bins:
+      - containerd
+
+  kubelet:
+    bins:
+      - containerd
+    defaultkubeconfig: /var/lib/rancher/k3s/agent/kubelet.kubeconfig
+    defaultcafile: /var/lib/rancher/k3s/agent/client-ca.crt
+
+  scheduler:
+    bins:
+      - containerd
+
+  controllermanager:
+    bins:
+      - containerd
+
+  etcd:
+    bins:
+      - containerd
+
+  node:
+    components:
+      - kubelet
+      - proxy
+
+    kubelet:
+      bins:
+        - containerd
+      defaultkubeconfig: /var/lib/rancher/k3s/agent/kubelet.kubeconfig
+      defaultcafile: /var/lib/rancher/k3s/agent/client-ca.crt
+
+    proxy:
+      bins:
+        - containerd
+      defaultkubeconfig: /var/lib/rancher/k3s/agent/kubeproxy.kubeconfig
+
+  policies:
+    components:
+      - policies
```

```suggestion
node:
    components:
      - kubelet
      - proxy

    kubelet:
      bins:
        - containerd
      defaultkubeconfig: /var/lib/rancher/k3s/agent/kubelet.kubeconfig
      defaultcafile: /var/lib/rancher/k3s/agent/client-ca.crt

    proxy:
      bins:
        - containerd
      defaultkubeconfig: /var/lib/rancher/k3s/agent/kubeproxy.kubeconfig

policies:
    components:
      - policies
```

Related to this: https://github.com/rancher/security-scan/pull/179/files#r1438362095
- kubelet conf
- fix indentation